prime-router/metadata/hl7_mapping/common/datatype/ce-coded-with-exceptions.yml

```diff
@@ -18,22 +18,22 @@ elements:
     hl7Spec: [ '%{cweFieldPath}-2' ]
   - name: identifier-system-url
     resource: '%resource.coding.where(extension(%`rsext-code-index-name`).value = "identifier").system'
-    condition: '%resource.value.startsWith("http")'
+    condition: '%resource.value.exists() and %resource.value.startsWith("http")'
```

Comment:
I guess `startsWith` has the same issue now as `is`?

Reply:
Yes, anything that returns a boolean is impacted